templates/es-client.yaml

```diff
@@ -97,7 +97,17 @@ spec:
         volumeMounts:
         - name: storage
           mountPath: /data
+        {{- if .Values.configmap.enabled }}
+        - name: cfgmap
+          mountPath: /mnt/cfgmap
```

Comment:
I rarely see /mnt as a target except for hostPath mounts

Reply:
I use `mnt` to mount configmap, it's just easier as the config map get `mounted` there. It will be much more intrusive mounting it in a system dir, as it can overwrite files you are unaware of